templates/server-acl-init-job.yaml

```
@@ -80,14 +80,18 @@ spec:
             - "/bin/sh"
             - "-ec"
             - |
+              CONSUL_FULLNAME="{{template "consul.fullname" . }}"
+
               consul-k8s server-acl-init \
-                -server-label-selector=component=server,app={{ template "consul.name" . }},release={{ .Release.Name }} \
+                {{- range $index := until (.Values.server.replicas | int) }}
+                -server-address=${CONSUL_FULLNAME}-server-{{ $index }}.${CONSUL_FULLNAME}-server.${NAMESPACE}.svc \
```

```suggestion
                -server-address="${CONSUL_FULLNAME}-server-{{ $index }}.${CONSUL_FULLNAME}-server.${NAMESPACE}.svc" \
```

Best practice to quote interpolated strings

---

💯